blazor/rich-text-editor/paste-cleanup.md

```diff
@@ -131,12 +128,11 @@ In the following example, the paste cleanup related settings are explained with
 {% endhighlight %}
 {% endtabs %}
 
-![Blazor RichTextEditor with Allowed Styles Paste CleanUp](./images/blazor-richtexteditor-paste-allowed-style.png)
-
+![Blazor RichTextEditor with allowed styles](./images/blazor-richtexteditor-paste-allowed-style.png)
 
 ## Pasting large text content
 
-When pasting a large text into the editor it displays "Attempting to reconnect" and then the text gets inserted. To archieve this, you can increase the SignalR size in application level by adding the ‘signalR’ method with larger buffer size(’1024000000’).
+When pasting a large text into the editor it displays "Attempting to reconnect" and then the text gets inserted. To achieve this, you can increase the SignalR size at the application level by adding the ‘signalR’ method with a larger buffer size(’1024000000’).
```

Comment:
Don't use double and single quotes in the UG. please correct it "Attempting to reconnect" and  ‘signalR’.  

Highlighted the above words.

Reply:
`Attempting to reconnect` and `signalR` need to high-light like this, right sir?